src/parseTools.js

```diff
@@ -1292,7 +1292,7 @@ function makeGetTempRet0() {
 }
 
 function makeSetTempRet0(value) {
-  return RELOCATABLE ? "setTempRet0((" + value + ") | 0)" : ("tempRet0 = " + value);
+  return 'asm["setTempRet0"](' + value + ')';
```

Comment:
No, as I said I simply don't know what to do - I don't know what relocatable in emscripten means. (I know what that means in traditional sense, but don't know how it applies to emscripten.) And I don't know what the difference is between setTempRet0(value) and tempRet0 = value. That's the reason why I wrote here
https://github.com/kripken/emscripten/pull/4524#issuecomment-243477623
that it's not that I'd like to commit this but ask how things work here.